### DISCUSSION TITLE
How to connect SULU to SMTP email?

### DISCUSSION THREAD

**Discussion by normenrollf** [sulu/SuluFormBundle#327] [Q&A] [answered]
Most content management systems (CMS) offer the possibility to set up the e-mail configuration to use SMTP instead of the PHP mail function. I could not find anything in the docs about either SMTP or the PHP mail function. How is the problem solved in Sulu?

Thx

**Comment by alexander-schranz** [comment, marked as answer]
Emails are handled in Sulu over Symfony. Currently it uses the `swiftmailer` package and so `SwiftmailerBundle` which you configure in your `.env` files (normally `.env.local`) via the `MAILER_URL`: https://github.com/sulu/skeleton/blob/74a98c4114ab5c7b19656b6604d86eacdbe47bb6/.env#L34

See also https://symfony.com/doc/4.4/reference/configuration/swiftmailer.html#transport

In the next minor release sulu will switch to the Symfony Mailer instead of Swiftmailer. Also there a `MAILER_DSN` will exist to configure how you want to send the emails.

**Reply by normenrollf** [reply]
Great @alexander-schranz ! Thx for your fast replay. If you could link this info to the FormBundle in the doc, it would certainly help some people in future ;)

**Reply by alexander-schranz** [reply]
Do you want to great a pull request for it Think it could be added to the `Config` part where the emails are configured: https://github.com/sulu/SuluFormBundle/blob/2.4/Resources/doc/index.md#config

**Reply by normenrollf** [reply]
Of course. I don't know it any differently from other cms. Will you add it or should I do it?

**Reply by alexander-schranz** [reply]
We are happy if you creating a pull request with adding the links to the documentation.
